kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/GetOrderPollingCall.java

```diff
@@ -35,6 +36,9 @@ private void getOrder(int pollingIndex) {
             if (pollingIndex < DELAY_SECONDS.length) {
                 Order order = remote.getOrderSync(orderID);
                 if (order == null || order.getStatus() == Status.PENDING) {
+                    if(order != null && pollingIndex == DELAYED_ATTEMPTED_NUMBER){
```

Comment:
if order == null should we also updated about Status.DELAYED?

Reply:
nop because the order is not exists